### ISSUE TITLE
Unable to focus to grid body after filtering 

### ISSUE THREAD

**Issue by limonte** [vaadin/vaadin-grid#1290]
### Live Demo

https://cdn.vaadin.com/vaadin-grid/5.0.3/demo/#grid-sorting-filtering-demos

### Steps to reproduce

1. Focus the third row.
2. Move focus to header and filter results so less than 3 results are shown in the body.
3. Try to move focus from the header to the body by pressing Tab.

### Expected outcome

It's possible to move focus from header to body with the keyboard.

### Expected outcome

It's not possible to move focus from header to body with the keyboard.
